pipelines/datasets/us_bls_cpi/constants.py

```
@@ -0,0 +1,73 @@
+    DATASET_ID = "us_bls_cpi"
+
+    # download.bls.gov 403s without a browser User-Agent; BLS asks for a contact
+    # email in the UA string.
+    BASE_URL = "https://download.bls.gov/pub/time.series"
+    USER_AGENT = (
+        "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 "
+        "(KHTML, like Gecko) Chrome/120 Safari/537.36 rdahis@basedosdados.org"
+    )
+
+    # survey dir -> survey label used in the data
+    SURVEYS = {"cu": "CPI-U", "cw": "CPI-W"}
```

Ruff (0.15.21)

[warning] Mutable default value for class attribute

(RUF012)